vcd/resource_vcd_lb_service_monitor_test.go

```diff
@@ -0,0 +1,167 @@
+// +build gateway lb lbServiceMonitor ALL functional
+
+package vcd
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+
+	"github.com/hashicorp/terraform/terraform"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccVcdLbServiceMonitor(t *testing.T) {
+
+	if vcdShortTest {
```

Comment:
This check should be moved after the call to `templateFill`, or else the test won't be generated with `make test-binary-prepare`

Reply:
Done.